app/javascript/mastodon/features/home_timeline/components/explore_prompt.jsx

```diff
@@ -0,0 +1,16 @@
+import { FormattedMessage } from 'react-intl';
+
+import { Link } from 'react-router-dom';
+
+import DismissableBanner from 'mastodon/components/dismissable_banner';
+
+export const ExplorePrompt = () => (
+  <DismissableBanner id='home.explore_prompt'>
+    <p><FormattedMessage id='home.explore_prompt' defaultMessage="Your home feed is all the people you follow and the posts they share. It becomes more useful when you follow more people. It looks like it's not very lively yet, so how about:" /></p>
```

Comment:
Home feed also includes posts from hashtags you follow, doesn't it?

Reply:
Yeah, but that level of detail is probably out of place?

Reply:
I think people are commonly confused by that, so if we're telling people something, it probably would be good to be accurate, especially as posts from followed hashtags don't get labelled as such when in the home timeline.